controllers/storagecluster/storageconsumer.go

```diff
@@ -91,8 +93,8 @@ func (s *storageConsumer) ensureCreated(r *StorageClusterReconciler, storageClus
 		)
 		resourceMap := util.WrapStorageConsumerResourceMap(data)
 		resourceMap.ReplaceRbdRadosNamespaceName(util.ImplicitRbdRadosNamespaceName)
-		resourceMap.ReplaceSubVolumeGroupName("csi")
-		resourceMap.ReplaceSubVolumeGroupRadosNamespaceName("csi")
+		resourceMap.ReplaceSubVolumeGroupName(subVolumeGroupName)
+		resourceMap.ReplaceSubVolumeGroupRadosNamespaceName(subVolumeGroupName)
```

Comment:
Don't we get this automatically generated if we don't specify anything?

Reply:
we get the default and the default isn't `csi` and so using the replace method.